tests/validators/array.test.ts

```diff
@@ -11,6 +11,18 @@ describe('ArrayValidator', () => {
 		expect(() => predicate.parse('Hello there')).toThrow(new ValidationError('ArrayValidator', 'Expected an array', 'Hello there'));
 	});
 
+	const invalidArray = [[123], [true], [{}], [[]], [null]];
+
+	test.each([invalidArray])('Given an array with value %s other than string THEN throws CombinedPropertyError', (input) => {
+		expect(() => predicate.parse(input)).toThrow(
```

Comment:
```suggestion
	const invalidArray = [123, true, {}, [], null];

	test.each(invalidArray)('Given an array with value %s other than string THEN throws CombinedPropertyError', (input) => {
		expect(() => predicate.parse(input)).toThrow(
```

Reply:
Here I want to pass `[123]` and the array is an array of string so it should throw combinedError.

Also If I just pass `invalidArray` then it'll pass `123`. If I keep the array as 

```js
const invalidArray = [[123], [true], [{}], [[]], [null]];
```

and pass 
```js
test.each(invalidArray)
```
then also jest makes the array 1D. Means it's still passing `123`. So this one is correct.

ref: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
I'm very well aware of table arguments, they're just very hard to read properly, not to mention also inconsistent with the rest of the tests.

Reply:
Your changes aren't passing the test. It won't throw a `CombinedError` instead it'll throw a `ValidationError` and `ValidationError` is added just above this test.